packages/styles/scss/components/modal/_modal.scss

```diff
@@ -155,7 +155,6 @@
   .#{$prefix}--modal-container {
     position: fixed;
     display: grid;
-    overflow: hidden;
```

Comment:
removed this so that tooltip is visible outside the modal

Reply:
I think we'll need to keep that to ensure the visual design remains consistent. I think instead we could set the `align` property to `left` to ensure it's not rendered outside the bounds of the modal.

Reply:
Yeah sure 👍